assets/css/custom.css

```
@@ -40,6 +40,12 @@
   color: #00316A;
 }
 
+.google-map {
+  position: relative;
+  top: 114px;
+  z-index: 99;
+}
+
```

Wrap the page with a div with `.contact-us` card. 

```suggestion
.contact-us .google-map {
  position: relative;
  top: 114px;
  z-index: 99;
}

@media only screen and (max-width: 991px) {
  .contact-us .google-map {
    top: 123px;
  }
}
```